### PR TITLE
Feature/warnings

### LINKED ISSUE #65: Check for Module Conflicts
Leverage Manifest+ `conflicts` to check if a module has any known conflicts, if so, display those on the Bug Reporter form.

https://foundryvtt.wiki/en/development/manifest-plus#conflicts

### LINKED ISSUE #68: Add 0.8.x warning
If we detect the user is on 0.8.x, display a warning that things are definitely not stable.

### PR DESCRIPTION
**Pull request by Ethck**
Resolves #65
Resolves #68

### CODE REVIEW

File: Module/main.js [hunk 1]
```diff
@@ -81,6 +107,9 @@ class BugReportForm extends FormApplication {
       module: this.module,
       submittedIssue: this.submittedIssue,
       useBugReporter: this.useBugReporter,
+      // if core version > 0.7.10 (like 0.8.X)
+      unsupportedCore: isNewerVersion("0.8.0", "0.7.10"),
```
Comment (akrigline) [inline Module/main.js]: one of these should be `game.data.version` I think?

File: Module/templates/bug-report.html [hunk 2]
```diff
@@ -13,6 +13,11 @@ <h2 class="flexrow">
 </div>
 
 {{#if useBugReporter}}
+  <!--If game version is supported, hide this -->
+  <div class="{{#if unsupportedCore}}{{else}}hidden{{/if}} notification warning">
```
Comment (akrigline) [inline Module/templates/bug-report.html]:
```suggestion
  {{#unless unsupportedCore}}<div class="notification warning">
```
No need to create this DOM element unless we're in an unsupported core version, which we know from the start via getData. `hidden` class is good for async flows which we don't necessarily know about from the getgo.

File: Module/templates/bug-report.html [hunk 3]
```diff
@@ -13,6 +13,11 @@ <h2 class="flexrow">
...
+    <label>0.8.X versions of Foundry are in ALPHA and are likely to be extremely unstable.</label>
+  </div>
```
Comment (akrigline) [inline Module/templates/bug-report.html:20]:
```suggestion
  </div>{{/unless}}
```

File: Module/templates/bug-report.html [hunk 4]
```diff
@@ -61,6 +66,28 @@ <h4 class="flexrow">
     </div>
   </div>
 
+  <div class="{{#if (bugs-isEmpty conflicts)}}hidden{{/if}}">
+    <label>This module is known to conflict with the following:
+    <ul>
+    {{#each conflicts as | conflict |}}
```
Comment (akrigline) [inline Module/templates/bug-report.html]: `as | conflict |` is unecessary, doesn't hurt anything but the scope of 'this' becomes what you're assigning to `conflict` by default within the `#each`. Could simplify this after removing the `as` by:
- `conflict.active` -> `active` 
- `conflict.name` -> `name`
- etc...

**Review by akrigline — CHANGES_REQUESTED** [review]

File: Module/main.js (hunk 1 above)
Comment (Ethck) [reply Module/main.js]: Yep. Used a constant when testing lol. Thanks, this is why we have reviews!

File: Module/templates/bug-report.html (hunk 2 above)
Comment (Ethck) [reply Module/templates/bug-report.html]: Totally forgot about # unless

File: Module/templates/bug-report.html (hunk 4 above)
Comment (Ethck) [reply Module/templates/bug-report.html]: Huh, didn't know that!

**Comment by Ethck** [comment]
All changes should be done!

**Review by akrigline — APPROVED** [review]
👨‍🍳 💋

**Pull request merged**